docs/azure/integrations-overview.md

```diff
@@ -0,0 +1,196 @@
+---
+title: .NET Aspire Azure integrations overview
+description: Overview of the Azure integrations available in the .NET Aspire.
+ms.date: 12/12/2024
+uid: integrations/azure-overview
+---
+
+<!--
+
+Somebody reading the overview is supposed to understand:
+
+- What happens when you add an azure hosing integration and run locally 
+  - What about emulators?
+  - What about redis? I see AddRedis and AddAzureRedis, why do we have 2? (same for SQL and Postgres)
+- How do I customize azure resources?
+- What are these azure resources doing under the covers (leave the specifics for each integration doc, not the overview)
+
+Understanding of the AzureBicepResource and AzureProvisioning resource:
+ 
+- Bicep is foundational to understanding how aspire integrates with Azure. You can write bicep in C# with the Azure.Provisioning libraries. Aspire azure integrations use those libraries under the covers.
+- How do users pass Aspire parameter resource to AspireProvisioning resources and how do generally set parameters on resources.
+- How do you get outputs from resources (GetOutput)
+- How do are secrets handled in resources.
+- What are the special parameters that the system provides for me.
+
+Ties back to deployment:
+
+- What happens when these resources are deployed?
+- How does these show up in the manifest?
+- What does azd do here?
+
+-->
+
+# .NET Aspire Azure integrations overview
+
+[Azure](/azure) is the most popular cloud platform for building and deploying [.NET applications](/dotnet/azure). The [Azure SDK for .NET](/dotnet/azure/sdk/azure-sdk-for-dotnet) allows for easy management and use of Azure services. .NET Aspire provides a set of integrations with Azure services. This article details some common aspects of all Azure integrations in .NET Aspire and aims to help you understand how to use them.
+
+## Add Azure resources
+
+All .NET Aspire Azure hosting integrations expose Azure resources and by convention are added using `AddAzure*` APIs. When you add these resources to your .NET Aspire app host, they represent an Azure service. The `AddAzure*` API returns an <xref:Aspire.Hosting.ApplicationModel.IResourceBuilder`1> where `T` is the type of Azure resource. These `IResourceBuilder<T>` (builder) interfaces provide a fluent API that allows you to configure the underlying Azure resource within the [app model](xref:dotnet/aspire/app-host#terminology).
+
+### Typical developer experience
+
+When your .NET Aspire app host contains Azure resources, and you run it locally (typical developer <kbd>F5</kbd> experience), the Azure resources are provisioned in your Azure subscription. However, they're not yet deployed. Instead, they're running locally in the context of your app host.
```

Comment:
> However, they're not yet deployed. However, they're not yet deployed.

This isn't correct. They are deployed and running in Azure. They are deployed to a subscription and resource group specified for your local app development.